modules/ossm-install-prerequisites.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/preparing-ossm-install.adoc
+
+[id="ossm-install-prerequisites_{context}"]
+= Meeting the {ProductName} prerequisites
+
+Before you can install {ProductName}, you must meet the following prerequisites:
+
+.Prerequisites
+
+* Possess an active {product-title} subscription on your Red Hat account. If you do not have a subscription, contact your sales representative for more information.
+* Install {product-title} version 3.11, or higher. For more information about the system and environment requirements, see the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/installing_clusters/install-config-install-prerequisites#system-requirements[OpenShift Container Platform documentation].
+* Install the version of the {product-title} command line utility (the `oc` client tool) that matches your {product-title} version and add it to your path. For example, if you have {product-title} 3.11 you must have the matching `oc` client version 3.11. For installation instructions, see the {product-title} https://docs.openshift.com/container-platform/4.1/cli_reference/administrator-cli-commands.html[Command Line Reference] document.
```

Comment:
It's odd that you're mixing references to 4.1 and 3.11.

Reply:
I cleaned this up.